### ISSUE TITLE
combat.py implementation

### ISSUE THREAD

**Issue by gokceneraslan** [epigenelabs/pyComBat#2]
Hi, just wanted to let you know about an existing Python implementation of combat by @brentp:

https://github.com/brentp/combat.py

which we are using already in Scanpy (https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.combat.html) for batch correction in scRNA-seq data.

Any ideas about the differences between the implementations? Have you considering making a benchmark between them?

**Comment by shiva309** [comment]
Hi,

I was wondering the same thing as @gokceneraslan. Would like to know differences between the implementations?
Thank you.

**Comment by abdelkaderbehdenna** [comment]
Hi @gokceneraslan,
Hi @shiva309, 

My apologies for the late reply! Thank you so much for reaching out to us, and pointing out this other implementation of ComBat in Python.

pyComBat was actually at first an internal project for Epigene Labs. After benchmarking and testing it, we realised that it could be useful for a wider audience. We thus decided to make it public (here on GitHub) and write an method article on it (preprint available on bioRxiv: https://doi.org/10.1101/2020.03.17.995431).

We tried to be as exhaustive as possible in our implementation, which, therefore, also includes a non-parametric method, even faster than the R implementation. 

Unfortunately, there has been a shift of priority in our operational schedule lately, but we will get our hands dirty with this benchmarking as soon as the opportunity allows.

Best wishes,
Kader
